src/snapred/ui/view/NormalizationCalibrationRequestView.py

```diff
@@ -41,4 +41,10 @@ def verify(self):
             raise ValueError("Please select a sample")
         if self.groupingFileDropDown.currentIndex() == 0:
             raise ValueError("Please select a grouping file")
+        if self.smoothingParameterField.text() == "":
+            raise ValueError("Please enter a smoothing parameter")
```

Comment:
Is this removing the slider bar?

Reply:
I dont think so?  This just prevents the user from proceeding without setting a val in that field, which would otherwise crash the ui